### PR TITLE
[BACKLOG-32465] Hadoop Clusters - Embed Site XML files with Cluster C…

### PR DESCRIPTION
**Pull request by tkafalas**
…onfiguration

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 1m 11s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
impl/cluster,kettle-plugins/browse,kettle-plugins/common/namedClusterBridge,kettle-plugins/hadoop-cluster/ui,legacy
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 1m 22s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
impl/cluster,kettle-plugins/browse,kettle-plugins/common/namedClusterBridge,kettle-plugins/hadoop-cluster/ui,legacy
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 6m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
impl/cluster,kettle-plugins/browse,kettle-plugins/common/namedClusterBridge,kettle-plugins/hadoop-cluster/ui,legacy
```
---
:ok_hand: All tests passed!
 
Tests run: **658**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/big-data-plugin/job/PR-2028/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 5m 52s
Build command:
```console
mvn clean verify -B -e -Daudit -amd -pl \
impl/cluster,kettle-plugins/browse,kettle-plugins/common/namedClusterBridge,kettle-plugins/hadoop-cluster/ui,legacy
```
---
:ok_hand: All tests passed!
 
Tests run: **658**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](http://wingman2.pentaho.net/job/projects/job/big-data-plugin/job/PR-2028/4/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

File: kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java [hunk 1]
```diff
@@ -532,13 +532,14 @@ private void installSiteFiles( Map<String, CachedFileItemStream> siteFileSource,
         if ( name.equals( KEYTAB_AUTH_FILE ) || name.equals( KEYTAB_IMPL_FILE ) ) {
           name = extractFileNameFromFullPath( siteFile.getValue().getName() );
         }
-
-        File destination = new File(
-          getNamedClusterConfigsRootDir() + fileSeparator + nc.getName() + fileSeparator + name );
-        destination.getParentFile().mkdirs();
-        try ( OutputStream fos = new FileOutputStream( destination ) ) {
-          siteFile.getValue().getCachedOutputStream().writeTo( fos );
+        InputStreamReader isReader = new InputStreamReader( siteFile.getValue().getCachedInputStream() );
```
Comment (mkambol) [inline kettle-plugins/hadoop-cluster/ui/src/main/java/org/pentaho/big/data/kettle/plugins/hadoopcluster/ui/endpoints/HadoopClusterManager.java]: ^ should that be in a try-with-resources?

**Pull request merged**